README.md

```diff
@@ -29,7 +29,7 @@ use foxbox_users::users_router::UsersRouter;
 use iron::prelude::*;
 
 fn main() {
-    let router = UsersRouter::new();
+    let router = UsersRouter::init();
```

Comment:
This change is due to: [new_ret_not_self](https://github.com/Manishearth/rust-clippy/wiki#new_ret_no_self)